workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/DealRegistrationList/DealRegistrationList.tsx

```diff
@@ -164,6 +173,45 @@ const DealRegistrationList = ({sort}: IProps) => {
 		}
 	};
 
+	const todayDate = new Date();
+	const formattedDate = todayDate.toISOString().slice(0, 10);
+
+	const rangeDataPicker = submittedDealsFilter
+		? {
+				end: formattedDate,
```

Comment:
why limit the max date filter to today?

Reply:
Acceptance Criteria: https://liferay.atlassian.net/browse/ISSD-1810